napari/_qt/_qapp_model/qactions/_view.py

```diff
@@ -113,6 +126,45 @@ def _get_current_tooltip_visibility() -> bool:
         keybindings=[{'primary': KeyMod.CtrlCmd | KeyMod.Alt | KeyCode.KeyP}],
         toggled=ToggleRule(get_current=_get_current_play_status),
     ),
+    Action(
+        id='napari.viewer.reset_view',
+        title=trans._('Reset Zoom'),
```

Comment:
I'm not sure "Reset Zoom" is the right name here, because it also resets the camera angles. So I propose:

```suggestion
        title=trans._('Reset View'),
```

Reply:
Yeah I do think that is better with the current behavior, thanks for catching it.
The other option would be something like `Fit to View` (this is Command-9 on macOS), which may be slightly more intuitive than Reset View.
Or that plus something like `Actual Size` which would be zoom = 1

Reply:
Hmm yeah I think if it's changing camera angles we shouldn't call it reset zoom. If there is a standard keybinding for "fit to view" I'd say that should be used for this current behaviour, and then `Reset Zoom` should be separate, and it does what it says on the box and just zooms out...

Reply:
But the current behavior isn't really `Fit to view` because it also rotates the camera...
So still need some zoom-only functions.

Let me check if there already is a viewer keybinding for Reset -- yup, Command/Control-R

So maybe we don't need Reset View (the current "Reset Zoom" behavior) in the menu? It's a viewer button (the home). Or we could add `Reset View` to the menu with that keybinding, but if the user changes the keybinding in napari settings it will be an issue 🫤